protocol/x/revshare/keeper/revshare_test.go

```diff
@@ -257,10 +257,10 @@ func TestValidateRevShareSafety(t *testing.T) {
 	for name, tc := range tests {
```

Comment:
Can we add some new test case that would've passed before, but now fail the new requirement?